instance/tests/api/test_openedx_appserver.py

```diff
@@ -271,8 +271,8 @@ def test_spawn_appserver_break_on_success(
         spawn_appserver(instance.ref.pk, mark_active_on_success=mark_active, num_attempts=4)
         self.assertEqual(mock_provision.call_count, 1)
         # TODO: update this when redis becomes the default
-        self.assertEqual(mocks.mock_provision_rabbitmq.call_count, 1)
-        self.assertEqual(mocks.mock_provision_redis.call_count, 0)
+        self.assertEqual(mocks.mock_provision_rabbitmq.call_count, 0)
```

Comment:
👍 I thought that I already did this flip. -- Maybe I did it on the GitLab fork? 🤔